core/src/database/migrations.cpp

```diff
@@ -955,5 +955,17 @@ namespace ledger {
             sql << "DROP TABLE cosmos_currencies";
 
         }
+
+        template <> void migrate<21>(soci::session& sql, api::DatabaseBackendType type) {
+           sql << "ALTER TABLE bitcoin_outputs ADD replaceable INTEGER DEFAULT 0";
+        }
+
+        template <> void rollback<21>(soci::session& sql, api::DatabaseBackendType type) {
+            // SQLite doesn't handle ALTER TABLE DROP
+            if (type != api::DatabaseBackendType::SQLITE3) {
+                sql << "ALTER TABLE bitcoin_outputs DROP replaceable";
```

Comment:
Can you use the trick we did before to alter the table even for `sqlite` ?
When we get completely rid of `sqlite` support we can take such a shortcut